lru/lru_test.go

```diff
@@ -31,21 +31,21 @@ type complexStruct struct {
 	simpleStruct
 }
 
-var getTests = []struct {
-	name       string
-	keyToAdd   interface{}
-	keyToGet   interface{}
-	expectedOk bool
-}{
-	{"string_hit", "myKey", "myKey", true},
-	{"string_miss", "myKey", "nonsense", false},
-	{"simple_struct_hit", simpleStruct{1, "two"}, simpleStruct{1, "two"}, true},
-	{"simeple_struct_miss", simpleStruct{1, "two"}, simpleStruct{0, "noway"}, false},
-	{"complex_struct_hit", complexStruct{1, simpleStruct{2, "three"}},
-		complexStruct{1, simpleStruct{2, "three"}}, true},
-}
-
 func TestGet(t *testing.T) {
+	var getTests = []struct {
```

Comment:
I'd switch this to a short assignment:
```suggestion
	getTests := []struct {
```

Reply:
Sure thing -- holdover from the old code, wonder why he did that when it was global.